source/unified-test-format/schema-1.18.json

```diff
@@ -0,0 +1,1070 @@
+                        "id",
+                        "database"
+                    ],
+                    "properties": {
+                        "id": {
+                            "type": "string"
+                        },
+                        "database": {
+                            "type": "string"
+                        },
+                        "bucketOptions": {
+                            "type": "object"
+                        }
+                    }
+                },
+                "thread": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "id"
+                    ],
+                    "properties": {
+                        "id": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "logComponent": {
+            "type": "string",
+            "enum": [
+                "command",
+                "topology",
+                "serverSelection",
+                "connection"
+            ]
+        },
+        "logSeverityLevel": {
+            "type": "string",
+            "enum": [
+                "emergency",
+                "alert",
+                "critical",
+                "error",
+                "warning",
+                "notice",
+                "info",
+                "debug",
+                "trace"
+            ]
+        },
+        "clientEncryptionOpts": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+                "keyVaultClient",
+                "keyVaultNamespace",
+                "kmsProviders"
+            ],
+            "properties": {
+                "keyVaultClient": {
+                    "type": "string"
+                },
+                "keyVaultNamespace": {
+                    "type": "string"
+                },
+                "kmsProviders": {
+                    "$ref": "#/definitions/kmsProviders"
+                }
+            }
+        },
+        "kmsProviders": {
+            "$defs": {
+                "stringOrPlaceholder": {
+                    "oneOf": [
+                        {
+                            "type": "string"
+                        },
+                        {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "required": [
+                                "$$placeholder"
+                            ],
+                            "properties": {
+                                "$$placeholder": {}
+                            }
+                        }
+                    ]
+                }
+            },
+            "type": "object",
+            "additionalProperties": false,
+            "patternProperties": {
+                "^aws(:[a-zA-Z0-9_]+)?$": {
```

Comment:
Noted that the named providers don't support automatic credentials, but I don't think we need to address that in the schema. If anything, allowing an empty object would be helpful if we want to test that a driver raises a runtime error.